packages/main/src/themes/Label.css

```diff
@@ -77,3 +77,9 @@ bdi {
 	margin-right: .125rem;
 	margin-left: 0;
 }
+
+.ui5-label-root:focus {
```

Comment:
:host([focused]) .ui5-label-root since the focused attribute is updated on focusin/focusout

Reply:
the label does not have a "focused" property... do you think we should add one, for consistency with other similar cases? 

btw I checked that some components e.g. Carousel also do not define a "focused" property and use the :focus instead